jekyll/_posts/guides/2016-10-18-e2e_implementation.rst

```diff
@@ -439,52 +367,17 @@ list of devices for each user in the room. This can be done proactively,
 or deferred until the first message is sent. The information is also
 required to allow users to `verify or block devices`__.
 
-__ `blocking`_
-
-The client should build a JSON query object as follows:
-
-.. code:: json
+__ `blocking`
```

Comment:
The last underscore here needs to be kept, or else the link breaks.  (I'm guessing it was deleted by accident.)